backend/src/index.ts

```diff
@@ -1,16 +1,64 @@
 import express, { Response } from "express";
 import session from "express-session";
-import { TypedRequest, LoginBody } from "./requestTypes";
+import { TypedRequest, LoginBody, ValidateBody } from "./requestTypes";
+import { SessionStorage, User } from "./interfaces";
 import bcrypt from "bcrypt";
-import { collection, addDoc, getDocs, where, query } from "firebase/firestore";
+import {
+  collection,
+  addDoc,
+  getDocs,
+  where,
+  query,
+  deleteDoc,
+} from "firebase/firestore";
 import { db } from "./firebase";
 import cors from "cors";
 import crypto from "crypto";
 
-const EXPRESS_PORT = 3000;
+const session_auth = async (sessionId: string): Promise<boolean> => {
```

Comment:
this function doesn't seem to return a promise but rather just a boolean?

Reply:
it says this 
> The return type of an async function or method must be the global Promise<T> type. Did you mean to write 'Promise<boolean>'?ts(1064)

so I'm unsure what to put

Reply:
i just removed the return types from them entirely 👍

Reply:
if it doesnt get mad then yeah that sounds good